src/zenml/steps/step_decorator.py

```diff
@@ -185,6 +178,14 @@ def inner_decorator(func: F) -> Type[BaseStep]:
         Returns:
             The class of a newly generated ZenML Step.
         """
+        step_name = name or func.__name__
+        logger.warning(
+            f"The `@step` decorator that you used to define your {step_name} "
+            "step is deprecated. Check out our docs https://docs.zenml.io for "
+            "information on how to define steps in a more intuitive and "
+            "flexible way!"
```

Comment:
I know you didn't write this, but should we: 1) link to something a bit more specific than just 'check out our docs' and b) make the message a bit less cutesy?

Reply:
I've updated that in my PR which now links to a migration guide